src/Compilers/CSharp/Portable/Binder/UsingStatementBinder.cs

```diff
@@ -149,6 +121,51 @@ internal override BoundStatement BindUsingStatementParts(DiagnosticBag diagnosti
                 boundBody,
                 awaitOpt,
                 hasErrors);
+
+            // returns true for error
+            bool initConversion(TypeSymbol disposableInterface, DiagnosticBag bag, bool fromExpression)
```

Comment:
>TypeSymbol disposableInterface, DiagnosticBag bag [](start = 32, length = 49)

Do we need this two parameters ```TypeSymbol disposableInterface, DiagnosticBag bag```? It looks like both call sites pas the same values.